draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1880,8 +1880,8 @@ Constants:
   3. -(3 * Z^2 + 4 * A) / (4 * g(Z)) is square in F.
   4. At least one of g(Z) and g(-Z / 2) is square in F.
 
-Sign of y: Inputs u and -u give the same x-coordinate for many values of u.
-Thus, we set sgn0(y) == sgn0(u).
+Sign of y: Inputs u and -u give the same x-coordinate for many values of u, so
+sgn0(y) == sgn0(u).
```

Comment:
The "we set" made clear that this was a decision, whereas the proposed wording feels like it implies that "sgn0(y) == sgn0(u)" follows from "u and -u give the same x-coordinate". Is it possible to recover the explicit we-made-this-decision statement?

Reply:
Yep, that's fine. I'll revert it.